### ISSUE TITLE
Add a __repr__ to `CNMFParams`?

### ISSUE THREAD

**Issue by zbarry** [flatironinstitute/CaImAn#526]
Would be nice just for quick sanity checking of params. Could be as simple as:

```python
class CNMFParams(object):

    ~~ snip ~~

    def __repr__(self):
        return str(self.to_dict())
```

Interested in something like this?

**Comment by epnev** [comment]
Sure, although it would be nice to print the subdictionaries one by one, and not as a single string

**Comment by epnev** [comment]
Addressed with #527
